.travis.yml

```diff
@@ -11,12 +11,25 @@ notifications:
     channels:
       - "chat.freenode.net#videojs"
     use_notice: true
-# Set up a virtual screen for Chrome.
 before_script:
+  # check if the current version is equal to the version for the env
+  - 'export IS_INSTALLED="$(npm list video.js | grep "video.js@$VJS")"'
```

Comment:
This will pick up `video.js@6` in `videojs-flash` and never install `video.js@6` at the top level. Perhaps instead use `npm list video.js --depth 0`